var/ramble/repos/builtin/applications/hostname/application.py

```diff
@@ -28,7 +29,10 @@ class Hostname(ExecutableApplication):
     workload('serial', executable='serial')
     workload('parallel', executable='parallel')
 
-    figure_of_merit('user time', log_file='{experiment_run_dir}/{experiment_name}.out',
+    log_str = (Expander.expansion_str('experiment_run_dir') + '/' +
```

Comment:
Should probably use `os.path.sep` instead of `/` (optionally, could use `os.path.join`)

Reply:
Done